content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -1,296 +1,425 @@
+7. In the Asset Bundler, navigate to the **Asset Lists** tab to verify the assets in `engine.assetlist`. The assets are listed in the **Asset List** panel.
 
 #### Bundle your assets
 
-1. Select your `engine.assetlist` in the** Asset List Files** panel under the **Asset Lists** tab.
+1. In the Asset Bundler, on the **Asset Lists** tab, in the **Asset List Files** panel, select the `engine.assetlist` file.
+
+2. Click **Generate Bundle**, which opens the **Generate Bundles** dialog.
+
+3. Click **Browse...**, which opens File Explorer.
+
+4. In File Explorer, enter a name for your package file, and then click **Save**. For this example, use the name `engine.pak`.
+
+
+For Windows, when Asset Bundler saves your bundle, it appends `_pc` to the bundle's name. So, you now have two asset bundles: `game_pc.pak` and `engine_pc.pak`.
+
 
-2. Click **Generate Bundle**. This opens the **Generate Bundles** dialog. 
+### Add bundles to the project game release layout
 
-3. Click **Browse...** to open the File Explorer. 
 
-4. Enter a name for your package file and save. In this example, we'll use the name `engine_bundle.pak`.
+Next, add your `game_pc.pak` and `engine_pc.pak` files to your project game release layout, so the Game Launcher can load assets from those bundles.
 
+1. Navigate to the `<install>/bin/Windows/release/<build>/Cache/pc` directory, which contains the bundled content in your project game release layout directory. For example, the default path may be:
 
-You now have two bundled content: `game_bundle.pak` and `engine_bundle.pak`. 
+   - `C:/MyProject/install/bin/Windows/release/Default/Cache/pc` -- For non-monolithic.
 
+   - `C:/MyProject/install/bin/Windows/release/Monolithic/Cache/pc` -- For monolithic.
 
-### Add your bundles to install layout
+2. Remove the `engine.pak` file that was created earlier in [Create a project game release layout](#create-a-project-game-release-layout). You don't need it because you created new bundles.
 
-Next, add `game_bundle.pak` and `engine_bundle.pak` to `install/bin/Windows/release/Default`. You can remove the `engine.pak` file that was created earlier. 
+3. Add your new bundles: `game_bundle_pc.pak` and `engine_bundle_pc.pak`.
 
 
 ## Run the game launcher
 
-Now that you've packaged your assets and built your project game release layout, you are ready to run your project's Game Launcher.
+Now you're ready to run your project's Game Launcher.
 
-1. Open a command line window and change to your project's release build install directory.
+Run `GameLauncher.exe` from your project game release layout, which is located in the `<install>/bin/Windows/release/<build>` directory. Examples of the path are
 
-    - For monolithic release builds:
-        ```cmd
-        cd C:/o3de-projects/MyProject/install/bin/Windows/release/Monolithic
-        ```
+- `C:/MyProject/install/bin/Windows/release/Default` -- for non-monolithic.
```

Comment:
```suggestion
- `C:\MyProject\install\bin\Windows\release\Default` -- For non-monolithic projects.
```
Again here, I'm unsure if "builds" or "projects" is the correct word to append here. Or something else. But use a noun to complete the thought here.